src/StackAdmin/Azs.Network.Admin/readme.md

```diff
@@ -141,5 +159,17 @@ directive:
       default:
         script: '50'
 
+    #rename properties
+  - where:
+      model-name: VirtualNetwork
+      property-name: ConfigurationStateHostError
+    set:
+      property-name: ConfigurationStateHostErrors
+  - where:
+      model-name: VirtualNetwork
+      property-name: ConfigurationStateVirtualNetworkInterfaceError
+    set:
+      property-name: ConfigurationStateVirtualNetworkInterfaceErrors
```

Comment:
I prefer taking a breaking change here, rather than doing a customization in the object model, i am merging for now. I will follow up with you offline